packages/frontend/src/routes/signIn/index.tsx

```diff
@@ -38,17 +37,18 @@ function SignIn() {
   const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(true);
 
   const userCode = params.get("code") ?? "";
-
   storeDataInCookie("userCode", userCode, 1);
 
   useEffect(() => {
     const userCodeFromCookie = getDataInCookie("userCode");
-
     const isGoogleInUrl = window.location.href.includes("google")
       ? "google"
       : "github";
 
+    console.log(userCodeFromCookie);
```

Comment:
Is there a particular reason for storing the loading in cookies? Maybe you can just add a loading state that you can toggle when making the request? 
Also can you remove the console.log

Reply:
On it!